core/src/main/java/org/lflang/generator/c/CMainFunctionGenerator.java

```diff
@@ -80,6 +80,10 @@ private String generateMainFunction() {
       case RP2040 -> {
         return String.join("\n", "int main(void) {", "   return lf_reactor_c_main(0, NULL);", "}");
       }
+
+      case STM32 -> {
+        return String.join("\n", "int main(void) {", "   return lf_reactor_c_main(0, NULL);", "}");
+      }
```

Comment:
Identical to RP2040, so merge it into the same case?

Reply:
I personally feel like its easier to read when we have the separation for STM32 and rp2040 though